.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Is this official template or new one?

Reply:
It is not an official template.
However, it is already applied to Chart, Editor and Grid of TOAST UI Product.
What do you think?

Reply:
What is the issue type? I think this is enough to fullfill a form.